docs/_sources/chapters/data-and-variables/values-and-data-types.rst.txt

```diff
@@ -1,80 +1,91 @@
 Values and Data Types
 =====================
 
-.. index:: ! value, data type, string, number, object, type, type system, console.log
+.. index:: ! value
 
-A **value** is a specific piece of data--like a word or a number--that a program manipulates. The values we have seen so far are ``5``, ``5.2``, and ``"Hello, World!"``.
+Data can be stored in a program in a variety of ways. The most basic unit of data is a value.
 
-.. index:: ! data type, ! number, ! string
+A **value** is a specific piece of data, such as a word or a number. Some examples are ``5``, ``5.2``, and ``"Hello, World!"``.
 
-Each value belongs to a category called a **data type**: ``4`` is an **number**, and ``"Hello, World!"`` is a **string**, so-called because it contains a string, or sequence, of letters. You (and the interpreter) can identify strings because they are enclosed in either single or double quotation marks.
+.. index:: ! data type, ! number, ! string, ! type 
 
-.. index:: ! integer, ! float
-
-Numbers may be either **integer**--such as 1, -15, or 42--or a **float**--such as 2.12 or 3.14. An integer is a whole number, with no decimal part. A float has a decimal part.
+Each value belongs to a category called a **data type**: ``4`` is a **number**, and ``"Hello, World!"`` is a **string**, so-called because it contains a string, or sequence, of letters. When using strings in a JavaScript program, we must enclose them in either single or double quotes. 
 
-.. warning:: While a number like ``3.0`` may seem like an integer because it's decimal part is 0, it is usually considered to be a float.
+.. index:: ! integer, ! float
 
 .. index:: ! typeof
 
-If you are not sure what data type a value falls into, JavaScript has an operator called ``typeof`` which can tell you.
+If you are not sure what data type a value falls into, precede the value with ``typeof``.
 
-.. code-block:: js
+.. sourcecode:: js
 
    console.log(typeof "Hello, World!");
    console.log(typeof 17);
    console.log(typeof 3.14);
 
-Not surprisingly, strings are of type ``'string'`` while integers and floats are of type ``'number'``. 
+**Output**
 
-.. note::
+::
 
-	When we show the value of a string using the ``console.log()`` function, such as with ``console.log("Hello, World!")``, the quotes are no longer present. The value of the string is the sequence of characters inside the quotes. The quotes are only necessary to help JavaScript know what the value is, and are not part of the value itself.
+   string
+   number
+   number
 
-.. index:: operator
+Not surprisingly, strings are of type ``string`` while integers and floats are of type ``number``. Note that some JavaScript environments may print types names and strings with single quotes around them, as in ``'string'``, ``'number'``, and ``'hello'``.
```

Comment:
@welzie This sentence has been improved. I included the last sentence because students can get confused if their output looks different from what we show them. In repl.it, data types are output w/o quote marks, while the local Node repl outputs data types with quote marks. Do you have a suggestion for improving the wording to make it more relevant / less confusing?

Reply:
Maybe mention repl.it and codio as the two environments? Or just ignore my comment, I may be getting too picky.

Reply:
It's a valid point. We will discuss different JS environments a bit in chapter 3, so they should have some additional context for this comment by the time they get to this chapter.